src/AWSS3.jl

```diff
@@ -824,6 +824,21 @@ end
 s3_sign_url(a...;b...) = s3_sign_url(global_aws_config(), a...;b...)
 
 
+"""
+    s3_nuke_bucket(bucket_name)
+
+This function is NOT exported on purpose.
```

Comment:
Hm why do we not want to export this? Can explain more in the docstring.

Reply:
Updated the docstr explaining. Just as a half-measure for people blowing up stuff by accident.